### PR TITLE
ocamlPackages.jingoo: 1.2.18 → 1.3.4

### PR DESCRIPTION
**Pull request by sternenseemann**
###### Motivation for this change

Update package, switch to dune build system.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

Also tested own project that depends on jingoo.

### CODE REVIEW

**Comment by sternenseemann** [comment]
@GrahamcOfBorg build ocamlPackages.jingoo

File: pkgs/development/ocaml-modules/jingoo/default.nix [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
     owner = "tategakibunko";
     repo = "jingoo";
     rev = "v${version}";
-    sha256 = "0gciiysrjy5r4yiisc41k4h0p530yawzqnr364xg8fdkk444fgkn";
+    sha256 = "0fsmm6wxa3axwbcgwdidik3drg754wyh2vxri2w12d662221m98s";
   };
 
-  buildInputs = [ ocaml findlib ];
-  propagatedBuildInputs = [ pcre uutf ];
+  propagatedBuildInputs = [ menhir ppx_deriving re uutf uucp ];
```
Comment (vbgl) [inline pkgs/development/ocaml-modules/jingoo/default.nix:17]: `menhir` does not need to be propagated.

Comment (sternenseemann) [reply pkgs/development/ocaml-modules/jingoo/default.nix]: fixed

**Comment by vbgl** [comment]
@GrahamcOfBorg build ocamlPackages.jingoo

**Pull request merged**